### PR TITLE
Remove stray trailing escapes

### PR DESCRIPTION
**Pull request by Rageking8**
Remove some trailing escapes as they are rendered as a stray backslash at the end of the link or sentence. This PR also contains some rudimentary drive-bys.

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@TylerMSFT

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/standard-library/chrono-operators.md [hunk 1]
```diff
@@ -1504,9 +1504,9 @@ The year and month.
 33\) `year_month_weekday(year(y), mwd.month(), mwd.weekday_indexed())`\
 34\) `year_month_weekday(y, mwd.month(), mwd.weekday_indexed())`\
 35\) `year_month_weekday(year(y), mwd.month(), mwd.weekday_indexed())`\
-36) `year_month_weekday_last(ym.year(), ym.month(), wdl)`\
+36\) `year_month_weekday_last(ym.year(), ym.month(), wdl)`\
 37\) `year_month_weekday_last(y, mwdl.month(), mwdl.weekday_last())`\
-38) `year_month_weekday_last(year(y), mwdl.month(), mwdl.weekday_last())`\
+38\) `year_month_weekday_last(year(y), mwdl.month(), mwdl.weekday_last())`\
```
[the remark this answers is not in the document]
Comment (TylerMSFT) [reply docs/standard-library/chrono-operators.md:1509]: I don't know why the backslashes are there. Let's get this PR through, and if it is something you want to do, removing them would be nice in a future PR.

**Review by TylerMSFT — APPROVED** [review]
Nice cleanup - thank you

**Review by TylerMSFT — APPROVED** [review]
Thank you for cleaning up the unsightly `\`

**Comment by TylerMSFT** [comment]
#sign-off

**Comment by TylerMSFT** [comment]
#sign-off

**Pull request merged**